modules/apps/translation/translation-web/test/js/import-translation/ImportTranslation.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+import {cleanup, fireEvent, render, within} from '@testing-library/react';
+import React from 'react';
+
+import ImportTranslation from '../../../src/main/resources/META-INF/resources/js/import-translation/ImportTranslation';
+
+const baseProps = {
+	errorMessage: '',
+	portletNamespace: '_mock_TranslationPortlet_',
+	portletResource: 'portletResource',
+	publishButtonLabel: 'Publish',
+	redirect: 'http://redirect-url',
+	saveButtonLabel: 'Save',
+	title: '',
+	workflowActionPublish: 1,
+	workflowActionSaveDraft: 2,
+	workflowPending: false,
+};
+
+const files = [...new Array(5)].map(
+	(_, i) => new File(['(っ◕‿◕)っ'], `translated_content_${i}.xlf`)
+);
+
+const renderComponent = (props) => render(<ImportTranslation {...props} />);
+
+describe('ImportTranslation', () => {
+	afterEach(cleanup);
+
+	describe('renders (default)', () => {
+		it('publish and save buttons are disabled', () => {
+			const {getByRole} = renderComponent(baseProps);
+
+			expect(
+				getByRole('button', {
+					name: baseProps.publishButtonLabel,
+				})
+			).toBeDisabled();
+
+			expect(
+				getByRole('button', {
+					name: baseProps.saveButtonLabel,
+				})
+			).toBeDisabled();
+		});
+
+		it('select file button is enabled and not remove files button rendered', () => {
+			const {getByRole, queryByRole} = renderComponent(baseProps);
```

Comment:
Better _select file button is enabled and remove files button is not rendered_ ?